MSAL/src/public/MSALDeviceInformation.h

```diff
@@ -37,7 +37,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  Device mode configured by the administrator
 */
-@property (nonatomic, readonly) MSALDeviceMode deviceMode;
+@property (nonatomic, readonly) MSALDeviceMode deviceMode API_AVAILABLE(ios(13.0), macos(10.15));
```

Comment:
I think it's fine to have this available everywhere. If we're below iOS 13, it would always return default mode.